chart/kubeapps/values.yaml

```diff
@@ -1538,3 +1538,226 @@ postgresql:
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+  ##
+  podLabels: {}
+  ## @param kubeappsapis.podAnnotations Annotations for KubeappsAPIs pods
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  podAnnotations: {}
+  ## @param kubeappsapis.podAffinityPreset Pod affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+  ##
+  podAffinityPreset: ""
+  ## @param kubeappsapis.podAntiAffinityPreset Pod anti-affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
+  ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+  ##
+  podAntiAffinityPreset: soft
+  ## nodeAffinityPreset Node affinity preset
+  ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
+  ##
+  nodeAffinityPreset:
+    ## @param kubeappsapis.nodeAffinityPreset.type Node affinity preset type. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
+    ##
+    type: ""
+    ## @param kubeappsapis.nodeAffinityPreset.key Node label key to match. Ignored if `affinity` is set
+    ##
+    key: ""
+    ## @param kubeappsapis.nodeAffinityPreset.values Node label values to match. Ignored if `affinity` is set
+    ## E.g.
+    ## values:
+    ##   - e2e-az1
+    ##   - e2e-az2
+    ##
+    values: []
+  ## @param kubeappsapis.affinity Affinity for pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ## NOTE: kubeappsapis.podAffinityPreset, kubeappsapis.podAntiAffinityPreset, and kubeappsapis.nodeAffinityPreset will be ignored when it's set
+  ##
+  affinity: {}
+  ## @param kubeappsapis.nodeSelector Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+  ## @param kubeappsapis.tolerations Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+  ## @param kubeappsapis.priorityClassName Priority class name for KubeappsAPIs pods
+  ##
+  priorityClassName:
+  ## @param kubeappsapis.hostAliases Custom host aliases for KubeappsAPIs pods
+  ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+  ##
+  hostAliases: []
+  ## kubeappsapis service parameters
+  ##
+  service:
+    ## @param kubeappsapis.service.port KubeappsAPIs service HTTP port
+    ##
+    port: 8080
+    ## @param kubeappsapis.service.annotations Additional custom annotations for KubeappsAPIs service
+    ##
+    annotations: {}
+## Redis&trade; chart configuration
+## ref: https://github.com/bitnami/charts/blob/master/bitnami/redis/values.yaml
+##
+redis:
```

Comment:
Not sure why we want to promote the options for redis in the values.yaml of the release, rather than just leaving them in the development values.kubeappsapis.yaml (as you've done) for now?

Reply:
Honestly, I didn't manually verify it but I assumed, given that many redis values are used in different places (notes.txt, chart.yaml for conditionally load redis, that I had to define the redis values in the chart even they are not used at all.
Indeed, we have to thoroughly review them when enabling it by default; I'm not sure the password is being set properly.

Another reason for doing so is that this way we can just use the values.yaml as is, without any more conflicts when syncing dev-prod charts.

Anyway, happy to spend some cycles in removing redis -though it would imply still having two separate chart versions; not sure if's the best option for us right now-. See what you think.